lib/github/api.rb

```diff
@@ -77,5 +110,29 @@ def parse_body(body)
 
       JSON.parse(body, symbolize_names: true)
     end
+
+    def get_next_page(header)
+      # GitHub pagination returns a "Link" header in the following format:
```

Comment:
I'm getting an error here, maybe the `res[:link]` doesn't always have a value.

```suggestion
      return unless header

      # GitHub pagination returns a "Link" header in the following format:
```